objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -718,6 +825,7 @@ public void testQuery() throws ImageProcessingException, InterruptedException {
             + "             \"anchors\":[0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828], "
             + "             \"pbFile\": \"" + PB_FILE + "\", "
             + "             \"labelFile\": \"" + LABEL_FILE + "\", "
+            + "             \"metaFile\": \"" + META_FILE + "\", "            
             + "             \"type\": \"yolo\""
             + "         }"
             + "}";
```

Comment:
We should update the VANTIQ project zip to be up to date -- defining only the metaFile & pbFile since that's the preferred method of operation these days.